griptape/tools/computer/tool.py

```diff
@@ -188,7 +187,7 @@ def build_image(self, tool: BaseTool) -> None:
 
     def dependencies(self) -> list[str]:
         with open(self.requirements_txt_path) as file:
-            return [line.strip() for line in file.readlines()]
+            return [line.strip() for line in file]
```

Comment:
Is this just a refactor or a change in behavior?

Reply:
https://docs.astral.sh/ruff/rules/readlines-in-for/